openedx/core/djangoapps/programs/signals.py

```diff
@@ -2,7 +2,6 @@
 This module contains signals / handlers related to programs.
```

Comment:
This is almost entirely linter improvements that I missed on my first past of lint-only changes, except for the 4 lines removed at the bottom.